includes/cart/functions.php

```diff
@@ -782,7 +791,19 @@ function edd_get_cart_fees( $type = 'all', $download_id = 0, $price_id = NULL )
  * @return float Total Cart Fees
  */
 function edd_get_cart_fee_total() {
-	return EDD()->fees->total();
+	$fees = EDD()->fees->get_fees( 'all' );
+
+	$fee_total = 0.00;
+	foreach ( $fees as $fee ) {
+		if ( ! empty( $fee['download_id'] ) && ( $fee['amount'] <= 0 || $fee['no_tax'] === true ) ) {
```

Comment:
Might have been an oversight, I'll test it out again, there is a scenario I was accounting for on fees with a download ID and that shouldn't affect taxes. I'll check it again.